src/hypothesis/core.py

```diff
@@ -662,6 +660,8 @@ def wrapped_test(*arguments, **kwargs):
 
 
 def find(specifier, condition, settings=None, random=None, database_key=None):
+    """Returns the minimal example from the given strategy ``specifier`` that
+    matches the predicate function ``condition``."""
```

Comment:
I also think it looks weird, but `pyformat` prefers same-line quotes for a single sentence.